chapter_preliminaries/ndarray.md

```diff
@@ -357,6 +395,14 @@ X_var[1, 2].assign(9)
 X_var
 ```
 
+```{.python .input}
+# JAX arrays are immutable
```

Comment:
Missing `#@tab jax`

Reply:
The syntax has changed, please use `%%tab jax`.